svf/include/SVFIR/SVFIR.h

```diff
@@ -393,6 +393,21 @@ class SVFIR : public IRGraph
     /// return whole allocated memory object if this node is a gep obj node
     /// return nullptr is this node is not a ObjVar type
     //@{
+    inline const SVFVar* getBaseVar(NodeID id) const
```

Comment:
Do we need this method?

Reply:
This method should be put in class SVFVar? as `SVFVar` only has two subclasses ValVar and ObjVar?